### PR TITLE
Stopwatch

### PR DESCRIPTION
**Pull request by Maksym845**
[DEMO LINK](https://Maksym845.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/stopwatch.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  &__minutes {
+    position: absolute;
+    left: 146px;
+    top: 75px;
+
+    height: 150px;
+    width: 9px;
+
+    animation: move;
```
Comment (volodymyrklymkiv) [inline src/styles/blocks/stopwatch.scss:41]: u can write this in one propety

File: src/styles/blocks/stopwatch.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  &__second {
+    position: absolute;
+    top: 25px;
+    left: 145px;
+
+    height: 250px;
+    width: 10px;
+
+    animation: move;
+    animation-duration: 60s;
```
Comment (volodymyrklymkiv) [inline src/styles/blocks/stopwatch.scss:61]: same

Comment (Maksym845) [reply src/styles/blocks/stopwatch.scss:61]: Really, I forgot about this, good tip!

**Review by volodymyrklymkiv — APPROVED** [review]
cool watch!

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
Your clock doesn't work

https://user-images.githubusercontent.com/92159085/187158463-77287f9e-7c2e-4c73-ade7-226704982d86.mov


Replace all magic numbers with variables

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**